backend/baseclass.pm

```diff
@@ -1241,6 +1242,8 @@ sub new_ssh_connection {
     }
 
     my $ssh = Net::SSH2->new;
+    # timeout requires libssh2 >= 1.2.9 so not all versions might have it
+    $ssh->timeout($bmwqemu::vars{SSH_COMMAND_TIMEOUT_S} * 1000) if $ssh->can('timeout');
```

Comment:
Now that you've changed it to `$bmwqemu::vars{SSH_COMMAND_TIMEOUT_S} * 1000`, won't this print a warning if the variable is `undef`?

Reply:
Argh, right, the `* 1000`, grmbl. Updated along with your other suggestion